LondonBritishLibrary/orient/BLorient12810.xml

```diff
@@ -0,0 +1,144 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BLorient12810" xml:lang="en" type="mss">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Ṭabiba ṭabibān</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                    Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS00001BL"/>
+                        <collection>Oriental</collection>
+                        <idno>BL Oriental 12810</idno>
+                        <altIdentifier><idno>Or. 12810</idno></altIdentifier>
+                        <altIdentifier><idno>Strelcyn 42</idno></altIdentifier>
+                    </msIdentifier>
+                    <msContents>
+                        <msItem xml:id="ms_i1">
+                            <locus from="1v" to="4v"/><locus from="54v" to="56r"/>
+                            <title ref="LIT1956Mashaf"/> 
+                            <incipit xml:lang="gez">በስመ፡ አብ፡ በል፡ መጽሐፈ፡ ኑዛዜ፡ መጽሐፈ፡ ኅድገተ፡ ኃቲአት፡ ወአበሳ፡ ስርዋ፡ ለጥበብ፡ ፈሪሃ፡ እግዚአብሔር፤ ስርዋ፡ ለኃጢአት፡
```

Comment:
ኃጢአት?

Reply:
It was quoted like that by Strelcyn:

<img width="327" alt="Screenshot 2024_Hatiat (II)" src="https://github.com/user-attachments/assets/909abe10-0373-4d5b-bf9b-aaa0dff78393">

I will add a sic.